src/recaptcha.js

```diff
@@ -174,6 +176,7 @@ ReCAPTCHA.propTypes = {
   stoken: PropTypes.string,
   hl: PropTypes.string,
   badge: PropTypes.oneOf(["bottomright", "bottomleft", "inline"]),
+  isolated: PropTypes.bool,
```

Comment:
❓ should we add a default prop value for `isolated`?
```
isolated: false
```

Reply:
I chose not to implement a default value because:
1. It is not a required prop, and
2. Google reCAPTCHA script defaults this to false anyway when not supplied - [see docs](https://developers.google.com/recaptcha/docs/invisible#render_param)

Open to discussion though!

Reply:
Sounds good